### PR TITLE
analysis message IST0150

### PR DESCRIPTION
**Pull request by eliavem**
Please provide a description for what this PR is for.
Analysis message for new issue type https://github.com/istio/istio/pull/36323 

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by linsun** [comment]
Nice work @eliavem! I validated the page is looking good - https://deploy-preview-10590--preliminary-istio.netlify.app/latest/docs/reference/config/analysis/ist0150/

@ericvn @rvennam PTAL, thanks!

**Review by linsun — APPROVED** [review]

File: content/en/docs/reference/config/analysis/ist0150/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: ExternalNameServiceTypeInvalidPortName
+layout: analysis-message
+owner: istio/wg-user-experience-maintainers
+test: no
+---
+
+This message occurs for services of type ExternalName when the port doesn't follow the [Istio service port naming convention](/docs/ops/deployment/requirements/), the port is unnamed or the port is named tcp.
```
Comment (ericvn) [inline content/en/docs/reference/config/analysis/ist0150/index.md:8]: Is this the best link? Maybe it should point the `Protocol Selection` page or maybe the `Explicit protocol collection` link.

File: content/en/docs/reference/config/analysis/ist0150/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    protocol: TCP
+    targetPort: 443
+  type: ExternalName
+{{< /text >}}
+
+In this example, the port name `tcp` follows the syntax: `name: <protocol>`. However, for ExternalName services, there is no service IP defined, so the SNI field is needed for routing.
+
+## How to resolve
+
+- If you have an ExternalName service type, and the protocol is TCP, rename the port to `<protocol>[-<suffix>]` or `<protocol>`  where protocol is HTTPS or TLS;
```
Comment (ericvn) [inline content/en/docs/reference/config/analysis/ist0150/index.md:39]: I think http and pls should be lowercase, at least they are in the table.

Comment (eliavem) [reply content/en/docs/reference/config/analysis/ist0150/index.md]: Hi @ericvn, lower case tls causes lint test to fail

File: content/en/docs/reference/config/analysis/ist0150/index.md (hunk 1 above)
Comment (eliavem) [reply content/en/docs/reference/config/analysis/ist0150/index.md]: added it

File: content/en/docs/reference/config/analysis/ist0150/index.md (hunk 2 above)
Comment (ericvn) [reply content/en/docs/reference/config/analysis/ist0150/index.md]: You probably need add backticks around them similar to earlier items in the sentence.

File: content/en/docs/reference/config/analysis/ist0150/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    protocol: TCP
+    targetPort: 443
+  type: ExternalName
+{{< /text >}}
+
+In this example, the port name `tcp` follows the syntax: `name: <protocol>`. However, for ExternalName services, there is no service IP defined, so the SNI field is needed for routing.
+
+## How to resolve
+
+- If you have an ExternalName service type, and the protocol is TCP, rename the port to `<protocol>[-<suffix>]` or `<protocol>` where protocol is https or TLS. To learn more, review
```
Comment (ericvn) [inline content/en/docs/reference/config/analysis/ist0150/index.md:39]:
```suggestion
- If you have an ExternalName service type, and the protocol is TCP, rename the port to `<protocol>[-<suffix>]` or `<protocol>` where protocol is `https` or `tls`. To learn more, review
```

**Review by ericvn — APPROVED** [review]

**Pull request merged**